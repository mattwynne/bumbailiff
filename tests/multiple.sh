#!/bin/bash
# shellcheck source=./test_helpers.sh
source "../../../test_helpers.sh"
export GIT_AUTHOR_NAME="Daisy Developer"
export GIT_AUTHOR_EMAIL="daisy@developer.com"
export GIT_AUTHOR_DATE="$(last_week)"
export GIT_COMMITTER_DATE="$(last_week)"
mkdir -p "git"
cd "git" || exit
git init --quiet .
git config commit.gpgSign false
git commit --quiet -m "Initial commit" --allow-empty
echo """// TODO: remove semicolon
alert('yo');
// TODO: extract method
""" >file.js
git add .
git commit --quiet --allow-empty -m "Add file.js"
export GIT_AUTHOR_DATE="$(yesterday)"
export GIT_COMMITTER_DATE="$(yesterday)"
echo """# TODO: rewrite in javascript
puts 'yo'
""" >file.rb
git add .
git commit --quiet --allow-empty -m "Add file.rb"
bumbailiff
echo $?
