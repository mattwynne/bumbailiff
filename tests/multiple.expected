[1mfile.js:1[0m
[0;32m7 days old[0;37m <a-commit-ref> (Daisy Developer)[0m
TODO: remove semicolon

[1mfile.js:3[0m
[0;32m7 days old[0;37m <a-commit-ref> (Daisy Developer)[0m
TODO: extract method

[1mfile.rb:1[0m
[0;32m1 days old[0;37m <a-commit-ref> (Daisy Developer)[0m
TODO: rewrite in javascript

‚õîÔ∏è [0;31m15 TODO-days accumulated (max allowance: 14 üíµ).[0m
1
